### PR TITLE
S3 t1 search jobs

### PR DESCRIPTION
**Pull request by j-add**

### CODE REVIEW

File: src/SearchJobs/SearchJobs.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
+        let noSpecialCharacters = str.replace(/[^a-zA-Z0-9 ]/g, '')
+        return noSpecialCharacters
+    }
+
+    useEffect(() => {
+        console.log(searchResults)
+        setJobData(searchResults)
+    }, [searchResults])
+
+    async function getSearchResults(searchQ) {
```
Comment (Parsley-Afroman) [inline src/SearchJobs/SearchJobs.js]: Something more semantic than SearchQ as this doesn't give insight to what it is

File: src/ToggleJobsView/ToggleJobsView.js [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
     const [jobsView, setJobsView] = useState(true)
 
+    async function getAllJobs() {
+        let response = await fetch('http://localhost:8080/jobs')
+        const jobData = await response.json()
+        setJobData(jobData)
+    }
+
     const handleClick = () => {
         setJobsView(!jobsView)
```
Comment (Parsley-Afroman) [inline src/ToggleJobsView/ToggleJobsView.js:15]: maybe some comment for readability for this

File: src/SearchJobs/SearchJobs.js [hunk 3]
```diff
@@ -1,12 +1,63 @@
...
-            <RecentJobs setModalShow={setModalShow} setJobId={setJobId} modalShow={modalShow} jobId={jobId} />
+            <div className="searchBanner">
+                <form>
+                    <h1>Find your perfect job</h1>
+                    <input type="text" onChange={handleChange}/>
+                    <button onClick={searchJobs}>Search</button>
+                </form>
+            </div>
+
+            <RecentJobs setModalShow={setModalShow} setJobId={setJobId} modalShow={modalShow} jobId={jobId} jobData={jobData} setJobData={setJobData} title={title} setTitle={setTitle}/>
```
Comment (ethan-garrett) [inline src/SearchJobs/SearchJobs.js]: could these props be laid out more neatly
like what how they've been laid out in recentJobs (lines 21-31)

**Review by ethan-garrett — CHANGES_REQUESTED** [review]
Other than the one requested change code looks good

**Review by Parsley-Afroman — APPROVED** [review]

**Pull request merged**